commoneventconsumer/src/main/scala/com/gu/notifications/events/model/EventAggregation.scala

```diff
@@ -1,11 +1,21 @@
 package com.gu.notifications.events.model
 
-import play.api.libs.json.{Json, Writes}
+import play.api.libs.json._
 
 
 case class EventAggregation(platformCounts: PlatformCount)
 
 object EventAggregation {
-  implicit val jreads = Json.reads[EventAggregation]
+  private val defaultJsonReads = Json.reads[EventAggregation]
+
+  implicit val jreads = new Reads[EventAggregation] {
+    override def reads(json: JsValue): JsResult[EventAggregation] = json match {
+      case JsObject(fields) =>
+        val platformCount = fields.get("platform").orElse(fields.get("platformCounts"))
```

Comment:
Do we need to support this indefinitely or just for the next few days / weeks?

Reply:
Indefinitely as the records in the database are already written like that. I just found it strange that the database model and the api model were the same except for that attribute name, adding these 250 lines of code for no good reason.